component/main.jsonnet

```
@@ -135,7 +140,7 @@ local configmap = kube.ConfigMap(params.configMapName) {
   },
   data: {
     'fluent-bit.conf': std.join('\n', [ service, filters, inputs, outputs ]),
-    'custom_parsers.conf': parsers,
+    'custom_parsers.conf': if std.length(parsers) > 0 || std.length(multiline_parsers) > 0 then std.join('\n', [ parsers, multiline_parsers ]) else '',
```

Assuming you use the suggestion above

```suggestion
    'custom_parsers.conf': if std.length(all_parsers) > 0 then all_parsers else '',
```